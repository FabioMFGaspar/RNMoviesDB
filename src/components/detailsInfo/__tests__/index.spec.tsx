import React from 'react';
import {render} from '@testing-library/react-native';
import DetailsInfo from '..';

describe('DetailsInfo test suite', () => {
  it('Default snapshot test', () => {
    const {toJSON} = render(
      <DetailsInfo title="dummyTitle" info="dummyOverview" />,
    );
    expect(toJSON()).toMatchSnapshot();
  });

  it('Should render title and overview', () => {
    const {queryByText} = render(
      <DetailsInfo title="dummyTitle" info="dummyOverview" />,
    );

    const title = queryByText('dummyTitle');
    const overview = queryByText('dummyOverview');

    expect(title).toBeTruthy();
    expect(overview).toBeTruthy();
  });
});
