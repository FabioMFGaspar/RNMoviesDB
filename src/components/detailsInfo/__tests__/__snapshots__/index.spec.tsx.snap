// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsInfo test suite Default snapshot test 1`] = `
<View
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "marginVertical": 2,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#8c8c8c",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
      }
    }
  >
    dummyTitle
  </Text>
  <Text
    style={
      Object {
        "color": "#8c8c8c",
        "flex": 1,
        "fontSize": 12,
      }
    }
  >
    dummyOverview
  </Text>
</View>
`;
